src/dune_rules/cram/cram_exec.ml

```diff
@@ -103,7 +102,7 @@ let translate_path_for_sh =
     match cygpath with
     | None -> User_error.raise [ Pp.text "Unable to find cygpath in PATH" ]
     | Some cygpath ->
-      Process.run_capture_line ~display:!Clflags.display Strict cygpath
+      Process.run_capture_line ~display:Quiet Strict cygpath
```

Comment:
This is a way of translating paths by calling cygpath. This is internal and does not need to be reported to the user.